pyfftw/pyfftw.pxd

```diff
@@ -47,6 +63,70 @@ cdef extern from 'pyfftw_complex.h':
     ctypedef long double clongdouble[2]
 
 cdef extern from 'fftw3.h':
+    """
```

Comment:
That's right, the block there should insert those raw macro definitions right after including fftw3 and those should keep the build working even without the actual functions being available, but they also shouldn't be called since they won't be added to the lookup tables since that's decided by the same conditions.